rediscluster/utils.py

```diff
@@ -203,3 +203,53 @@ def parse_slots(s):
         nodes.append(node)
 
     return nodes
+
+
+def parse_pubsub_channels(command, res, **options):
```

Comment:
I could move them to `client.py`, which is where they are in redis-py. 

I'm avoiding placing this logic in the methods themselves as that removes any ability to use `set_response_callback` to change the command's behaviour. In most cases as well, `execute_command` should return a parsed/correct response as it does with all other commands parsed by callback.